README.md

```diff
@@ -41,6 +41,8 @@ Parameters to the squid class almost map 1 to 1 to squid.conf parameters themsel
 * `coredump_dir` defaults to undef. [coredump_dir docs](http://www.squid-cache.org/Doc/config/coredump_dir/).
 * `max_filedescriptors` defaults to undef. [max_filedescriptors docs](http://www.squid-cache.org/Doc/config/max_filedescriptors/).
 * `workers` defaults to undef. [workers docs](http://www.squid-cache.org/Doc/config/workers/).
+* `acls` defaults to undef. [acl entries](http://www.squid-cache.org/Doc/config/acl/).
```

Comment:
I would maybe expand of the description of this argument as unlike the one to one mappings it is a lot less obvious. Maybe add an example or add to the existing example.